website/conferences/views.py

```diff
@@ -230,12 +229,7 @@ def conference_submissions_sql(conf):
                 LIMIT 1
               ) DOWNLOAD_COUNT ON TRUE
             -- Get all the nodes for a specific meeting
-            WHERE (osf_abstractnode_tags.tag_id IN
-                   (SELECT U0.id AS Col1
-                    FROM osf_tag U0
-                    WHERE (U0.system = FALSE
-                           AND UPPER(U0.name :: TEXT) = UPPER(%s)
-                           AND U0.system = FALSE))
+            WHERE (osf_conference_submissions.conference_id = %s
```

Comment:
Modified legacy query as this will go out before front end is waffled on, and legacy piece needs to work temporarily.  (The presence of a tag on a Node no longer adds it to a Conference; the Conference and the Node need to be related in the db.)